pkg/storage/mongo/getter.go

```diff
@@ -3,22 +3,25 @@ package mongo
 import (
 	"context"
 	"io"
+	"io/ioutil"
 
 	"github.com/globalsign/mgo"
-	"github.com/globalsign/mgo/bson"
 	"github.com/gomods/athens/pkg/errors"
 	"github.com/gomods/athens/pkg/observ"
 	"github.com/gomods/athens/pkg/storage"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/gridfs"
+	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 // Info implements storage.Getter
 func (s *ModuleStore) Info(ctx context.Context, module, vsn string) ([]byte, error) {
 	const op errors.Op = "mongo.Info"
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
-	c := s.s.DB(s.d).C(s.c)
+	c := s.client.Database(s.db).Collection(s.coll)
 	result := &storage.Module{}
-	err := c.Find(bson.M{"module": module, "version": vsn}).One(result)
+	err := c.FindOne(context.Background(), bson.M{"module": module, "version": vsn}).Decode(result)
 	if err != nil {
 		kind := errors.KindUnexpected
 		if err == mgo.ErrNotFound {
```

Comment:
Will this work? We should not use `github.com/globalsign/mgo` at all, no?
Does mongo-drive define it's own errors? It would be good to know if it was found or not.